api/swagger.yml

```diff
@@ -3783,3 +3810,29 @@ paths:
           $ref: "#/components/responses/NotFound"
         default:
           $ref: "#/components/responses/ServerError"
+
+  /statistics:
+    post:
+      tags:
+        - statistics
+      operationId: sendStatsEvents
+      summary: send stats events
```

Comment:
If it is not too much - can we call it postStatsEvents - old convention: usually send means it will happen now and post means it will be. (will have to address it also in the collector API)
but I think it will make the caller more aware when to expect it.

Reply:
done